webpages/js/metaanalyses.js

```diff
@@ -626,69 +673,116 @@
     }
 
     var currY = parseInt(plotEl.dataset.startHeight);
+    var currGY = 10;
```

Comment:
instead of 10 you probably want something like on the line above, put the value 10 in a data attribute on the template